package.json

```diff
@@ -28,7 +28,7 @@
     "pre-commit": "^1.2.2",
     "prettier": "^1.4.4",
     "raw-loader": "^0.5.1",
-    "serverless": "^1.13.1",
+    "serverless": "1.23.0",
```

Comment:
Could having a fixed value here interfere with our users' dependencies?

Reply:
This is a devDep so I don't think so?

Reply:
true true